tests/sentry/utils/performance_issues/test_render_blocking_asset_detector.py

```diff
@@ -18,6 +19,47 @@
     run_detector_on_data,
 )
 
+hash_config = load_span_grouping_config()
+
+
+def _valid_render_blocking_asset_event(url: str) -> Event:
+    event = {
+        "event_id": "a" * 16,
+        "project": PROJECT_ID,
+        "measurements": {
+            "fcp": {
+                "value": 2500.0,
+                "unit": "millisecond",
+            }
+        },
+        "spans": [
+            create_span(
+                "resource.script",
+                desc=url,
+                duration=1000.0,
+                data={
+                    "Transfer Size": 1200000,
+                    "Encoded Body Size": 1200000,
+                    "Decoded Body Size": 2000000,
+                },
+            ),
+        ],
+        "contexts": {
+            "trace": {
+                "span_id": "c" * 16,
+            }
+        },
+        "transaction": "/",
+    }
+    hash_config.execute_strategy(event).write_to_event(event)
```

Comment:
🤔 now that you're doing your own hashing, why do you need to use a real hashing strategy?

Reply:
Thanks, good catch! You're right, I don't need this anymore now that I've stopped using upstream span hashes at all.